### DISCUSSION TITLE
Replacing the Darksky API with Pirateweather API?

### DISCUSSION THREAD

**Discussion by natebc** [poblabs/weewx-belchertown#857] [Q&A] [answered]
With the imminent shutdown of the DarkSky API there's been some activity with alternatives. Just today on [hacker news](https://news.ycombinator.com/item?id=34329988) there was a discussion of a newish service that is _API compatible_ with DarkSky called [Pirate Weather](https://pirateweather.net/)

It seems like it would be a good drop in replacement for DarkSky once [March 31st](https://darksky.net/dev) rolls around.

**Comment by poblabs** [comment]
It could be, though I think most of us have moved to aeris weather which is
available in the latest release version as well as master.

I'm happy to look at PRs though if you want to give it a try.

**Comment by steepleian** [comment, marked as answer]
My first impressions are anything but good. The first hurdle is that they have a daily email limit which makes it impossible to register if the limit is exceeded. AerisWeather which is already coded into Belchertown is very solid and reliable with an impressive array of options with a free developers account.On 11 Jan 2023, at 15:38, Nate Childers ***@***.***> wrote:﻿
With the imminent shutdown of the DarkSky API there's been some activity with alternatives. Just today on hackernews there was a discussion of a newish service that is API compatible with DarkSky called Pirate Weather
It seems like it would be a good drop in replacement for DarkSky once March 31st rolls around.

**Comment by natebc** [comment]
I suspect the registration limit is due to them hitting the front of HN (and probably reddit but I haven't looked over there). If you guys aren't interested because of that and the inclusion of another forecasting service, then I suppose it's not worth the effort.
